mobile/App.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { StyleSheet, View } from 'react-native';
+import { Header, ImageContainer, ButtonsWrapper, Comments } from './components';
```

Comment:
This wont work because you are not adding and exporting the comments.  The first 3 are fine but you either need to bring the comments into your index.js and export it or do the comments on it's own line otherwise this was a great use of bringing everything into the index.js of your components folder and exporting them so you could do them in one line